docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -588,6 +588,18 @@ _Properties_
 -   _isDisabled_ `boolean`: Whether or not the user should be prevented from inserting this item.
 -   _frecency_ `number`: Heuristic that combines frequency and recency.
 
+### getLastFocus
```

Comment:
Can we avoid this being a public API please. (and the action as well)

Reply:
✅ https://github.com/WordPress/gutenberg/pull/57557